ruby/rails/security/audit/xss/avoid-link-to.yaml

```diff
@@ -0,0 +1,21 @@
+rules:
+- id: avoid-link-to
+  metadata:
+    owasp: 'A6: Security Misconfiguration'
+    cwe: "CWE-279: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')"
+    references:
+    - https://brakemanscanner.org/docs/warning_types/link_to/
+    - https://brakemanscanner.org/docs/warning_types/link_to_href/
```

Comment:
link_to and link_to_href are different errors in brakeman, but I didn't know which one to follow more closely because we don't check ruby versions. In the end I biased towards laziness, but we could mimic some of their strategies to avoid false positives. Interesting discussion: https://github.com/presidentbeef/brakeman/pull/45

Reply:
Updated to follow `link_to_href` more closely, still probably missing some cases. Message should possibly also be changed?